Shared/ContextMenuController.cpp

```diff
@@ -263,43 +259,33 @@ void ContextMenuController::setResultTitle(const QString& resultTitle)
  */
 void ContextMenuController::processGeoElements()
 {
-  if (m_contextFeatures.isEmpty() && m_contextGraphics.isEmpty())
+  if (m_contextGeoElements.isEmpty())
     return;
 
   // if we have at least 1 GeoElement, we can identify
   addOption(IDENTIFY_OPTION);
 
-  int pointGraphicsCount = 0;
-  for (const auto& geoElements : std::as_const(m_contextGraphics))
+  quint8 pointGeoElementCount = 0;
```

Comment:
Could there be more than 255 geoelements? Probably not, but the size savings may not be worth the risk of an overflow in an edge case.